Source/FieldSolver/FiniteDifferenceSolver/EvolveE.cpp

```diff
@@ -262,6 +265,11 @@ void FiniteDifferenceSolver::EvolveECylindrical (
         Array4<Real> const& jt = Jfield[1]->array(mfi);
         Array4<Real> const& jz = Jfield[2]->array(mfi);
 
+#ifdef AMREX_USE_EB
+        amrex::Array4<amrex::Real> const& lx = edge_lengths[0]->array(mfi);
```

Comment:
In the Ohm's law solver I used `lr` rather than `lx` to make it clear that this is actually an edge length in r. It could be nice to have the same convention between the two solvers but of course it is also not really needed.

Reply:
I won't merge yet and leave it up to you if you want to rename `lx` -> `lr` 😄

Reply:
Good point. I will modify the PR accordingly.